src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUE = 20;
+    private static final int MAX_VALUE_OF_DEFAULT_FIGURE = 10;
+    private static final String COLOR_OF_DEFAULT_FIGURE = "White";
```

Comment:
```suggestion
    private static final String DEFAULT_COLOR = "White";
```

get white from enum Color